site/docs/foundations/data-visualization/chart-accessibility.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Chart accessibility
+layout: DetailTechnical
+sidebar:
+  priority: 5
+---
+
+Charts rely heavily on visuals to help users understand complex data, so it’s important to ensure that your charts meet all accessibility requirements. Use color appropriately, include visual cues, provide text for assistive technologies, and ensure any interactions are accessible.
+
+## Categorical colors
+
+### Color contrast
+
+Users with visual impairments should be able to fully comprehend data sets that are represented by different colors. If you use custom colors or Hex codes, they may not meet non-text contrast requirements so you will be responsible to ensure that the resulting contrast is compliant. By using Salt tokens correctly, you will achieve at least 3:1 contrast between the background and other non-text chart elements such as bars, lines, segments and icons.
+
+<Image
+  src="/img/foundations/dataviz-hex.png"
+  srcDark="/img/foundations/dataviz-hex-dark.png"
+  alt="Diagram of best practices using categorical color versus color picking."
+/>
+
+### Color order
+
+Apply the categorical colors in the recommended order, from 1 through 20. 
+
+<Image
+  src="/img/foundations/dataviz-color-order.png"
+  srcDark="/img/foundations/dataviz-color-order-dark.png"
+  alt="Diagram of the categorical color palette."
+/>
+
+A well-defined, distinct color palette ensures that everyone can easily interpret the data. Using the set categorical palette colors minimizes the risk of misinterpretation and ensures a visual order. Don’t use shades of the same color.
+
+<Image
```

Comment:
Will this be contradicted by Heatmap? Would it help to include a second 'do not do this' example, for instance, applying very similar colors to the pie chart e.g. categories 3, 9, 13, 19.

Reply:
Would it also strengthen the argument if the white border was removed from the purple pie?

Reply:
Could we change the repeated 'label' to say something more relevant? Apples, Pears, Bananas? Manufacturing, Retail, Hospitality?